edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java

```diff
@@ -198,7 +199,7 @@ public void updateVariables(IProject project) throws CoreException {
 					String fileNameSplit[] = file.getName().split("[.]");
 					if(fileNameSplit[fileNameSplit.length-1].equalsIgnoreCase("jar"))
 					{
-						IPath filePath = new Path(file.getAbsolutePath());
+						IPath filePath = new Path("USERLIBS_DIR/" + file.getName());
```

Comment:
Fortunately it only does expansion to the first path segment (e.g. up to the first `/`).